src/pages/projectSettings/getTabTitle.ts

```diff
@@ -33,6 +33,9 @@ export const getTabTitle = (
       [ProjectSettingsTabRoutes.PeriodicBuilds]: {
         title: "Periodic Builds",
       },
+      [ProjectSettingsTabRoutes.Plugins]: {
+        title: "Plugins & Annotations",
+      },
```

Comment:
The designs have "Plugins" last in the sidebar, just wanted to ensure this ordering is intentional!

Reply:
It felt more natural to me to have Event Log last